src/components/calcite-tree/calcite-tree.e2e.ts

```diff
@@ -254,7 +254,6 @@ describe("calcite-tree", () => {
         await item1.click();
 
         expect(await getSelectedIds()).toEqual(["1"]);
```

Comment:
To improve our current coverage, can you add a test case for the `multi-children` selection mode?